src/main/java/org/folio/circulation/domain/ConfigurationRepository.java

```diff
@@ -62,6 +72,16 @@ private LoanAnonymizationConfiguration getFirstConfiguration(Collection<Configur
     return LoanAnonymizationConfiguration.from(new JsonObject(period));
   }
 
+  private Result<Response> refuseWhenNoConfigurationsFound(Response response) {
+    if (response.getJson().getInteger("totalRecords") == 0){
+      logger.warn("No loan history settings found for loan anonymization");
```

Comment:
Is this a warning? Is it ok for there not to be any configuration?

Reply:
I think so. Loan anonymization just exists with warning when there's no configuration. Nothing happens. It could be also be info...

Reply:
If no configuration is legitimate, and only for information or a warning at most, then should the anonymization endpoint respond with an error when the configuration is missing?

Reply:
The initiator of automatic loan anonymization is OKAPI and the receiver of the response is... OKAPI. I doubt that it makes any use of the response.

Reply:
I understand that. I think the response should be consistent irrespective of the client. If no configuration is acceptable, then the API should not respond with a client error.